cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -499,3 +499,454 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
+        expectedResults:
+            1. Notification for failure when creating the snapshot appears
+            2. Notification for failure when deleting the snapshot appears
+    Bugzilla:
+        1581793
+
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_notification_for_snapshot_actions_on_openstack():
+    """
+    Test task notification for snapshot tasks: success of create
+    and delete snapshot.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/3h
+        title: Test notification for snapshot actions on OpenStack
+        setup:
+            1. Add RHOS provider
+            2. Create a test VM on RHOS provider
+        testSteps:
+            1. Create a snapshot
+            2. Delete the snapshot
+        expectedResults:
+            1. Snapshot created successfully; notification displayed
+            2. Snapshot deleted successfully; notification displayed
+    Bugzilla:
+        1429313
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_sui_create_snapshot_from_vm_details_page_snapshot_page_and_service_details_page():
+    """
+    Snapshot can be created from VM details page, service details page
+    and snapshot page.
+    Check all pages and the snapshot count displayed on vm details page.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: SelfServiceUI
+        caseimportance: medium
+        initialEstimate: 1/4h
+        startsin: 5.9
+        title: SUI: Create snapshot from vm details page, snapshot page
+               and service details page
+        setup:
+            1. Add a vmware type provider
+            2. Create and order a service catalog which provisions a VM
+            3. Login to SSUI
+        testSteps:
+            1. Test that snapshot can be created from Service Details page
+                * Navigate to My Services -> <service_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            2. Test that snapshot can be created from VM Details page
+                * Navigate to My Services -> <service_name> -> <vm_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            3. Test that snapshot can be created from Snapshot page
+                * Navigate to My Services -> <service_name> -> <vm_name> -> Snapshots
+                * Click Configuration -> Create Snapshot
+                * Fill Name and click Create
+        expectedResults:
+            1. Snapshot successfully created
+            2. Snapshot successfully created
+            3. Snapshot successfully created
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_notification_for_snapshot_delete_failure_ec2():
```

Comment:
Have to use an EC2 provider marker to collect this test properly - and again move it to a module for cloud snapshot tests.

Reply:
Moved to /cloud/test_snapshot.